src/ddmd/expressionsem.d

```diff
@@ -1372,7 +1375,6 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
                         return;
                     }
                 }
-                break;
```

Comment:
Please explain why this is removed?

Reply:
Please see [this comment](https://github.com/dlang/dmd/pull/6439#discussion_r96134331) by @ZombineDev 

> It's very unusual that outer symbols are not visible in inner scopes (even if the symbol is rewritten to `opDispatch`).

Consider this case, and see the comment about `f2()`:

```D
class Foo
{
    void opDispatch(string name)() { pragma(msg, "Foo"); }
}
class Bar
{
    void opDispatch(string name)() { pragma(msg, "Bar"); }
}
class Baz
{
}

void main()
{
    auto foo = new Foo;
    auto bar = new Bar;
    auto baz = new Baz;

    with (foo)
    {
        f0();
        with (bar)
        {
            f1();
        }
        with (baz)
        {
            f2();   // Current Implementation:  Error: Error: undefined identifier f2
                    //  `opDispatch` is not implemented in `Baz`, but it is implemented
                    // in `Foo`.  Since we are still in `Foo`'s scope, it should use `Foo`'s
                    // `opDispatch`.
        }
    }
}
```